app/Http/Controllers/ReleaseController.php

```diff
@@ -40,11 +48,21 @@ public function show(Request $request)
     {
         $release = explode('.', $request->release);
 
+        $validator = Validator::make([
+            'major' => $release[0],
+            'minor' => $release[1] ?? null,
+            'release' => $release[2] ?? null,
+        ], [
+            'major' => ['required', 'regex:/^[\',"]?[5-9]|1[0-1][\',"]?$/'],
+            'minor' => ['nullable', 'regex:/^[\',"]?[0-9]{1,3}[\',"]?$/'],
+            'release' => ['nullable', 'regex:/^[\',"]?[0-9]{1,3}[\',"]?$/'],
```

Comment:
I opted for regex here since the major/minor/release values are strings, but I wanted to make sure they were reasonable integer values.